src/components/Calculator.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calc-container">
+        <p className="result-display">0</p>
+        <div className="keys-container">
+          <button className="grey" type="button">
+            AC
+          </button>
+          <button className="grey" type="button">
+            +/-
+          </button>
+          <button className="grey" type="button">
+            %
+          </button>
+          <button className="orange" type="button">
+            ÷
+          </button>
+          <button className="grey" type="button">
+            7
+          </button>
+          <button className="grey" type="button">
+            8
+          </button>
+          <button className="grey" type="button">
+            9
+          </button>
+          <button className="orange" type="button">
+            x
+          </button>
+          <button className="grey" type="button">
+            5
+          </button>
+          <button className="grey" type="button">
+            6
+          </button>
+          <button className="grey" type="button">
+            7
+          </button>
+          <button className="orange" type="button">
+            -
+          </button>
+          <button className="grey" type="button">
+            1
+          </button>
+          <button className="grey" type="button">
+            2
+          </button>
+          <button className="grey" type="button">
+            3
+          </button>
+          <button className="orange" type="button">
+            +
+          </button>
+          <button className="bigger grey" type="button">
+            0
+          </button>
+          <button className="grey" type="button">
+            .
+          </button>
+          <button className="orange" type="button">
+            =
+          </button>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Calculator;
```

Comment:
- [ ] Please your app is still rendering the `boiler plate`, created in the last milestone, Please kindly update your `App.js` file and import  your `Calculator component` into it and make sure it is rendering as specified in the design. Checkout the screenshot below for more clarifications. `Remember that App.js component is class-based`

![classbased](https://user-images.githubusercontent.com/62849872/153075653-014eef84-f655-43af-b74a-2baa39278717.png)

Reply:
hi @t-yanick, I can't figure out what this one is about, I exported and imported the class, and everything is a class-based component, I will be very pleased if you could explain me what the problem is :face_with_spiral_eyes: